arctic/store/_ndarray_store.py

```diff
@@ -150,27 +150,12 @@ def _index_range(self, version, symbol, from_version=None, **kwargs):
             from_index = from_version['up_to']
         return from_index, None
 
-    def get_info(self, arctic_lib, version, symbol, **kwargs):
-        collection = arctic_lib.get_top_level_collection()
-        dtype = self._dtype(version['dtype'], version.get('dtype_metadata', {}))
-        length = int(version['up_to'])
-
-        spec = {'symbol': symbol,
-                'parent': version.get('base_version_id', version['_id']),
-                'segment': {'$lt': length}}
-
-        n_segments = collection.find(spec).count()
-
-        est_size = dtype.itemsize * length
-        return """Handler: %s
-
-dtype: %s
-
-%d rows in %d segments
-Data size: %s bytes
-
-Version document:
-%s""" % (self.__class__.__name__, dtype, length, n_segments, est_size, pprint.pformat(version))
+    def get_info(self, version):
+        ret = {}
+        ret['dtype'] = version['dtype']
```

Comment:
no, because the dtype of a numpy array will more than likely be a string like float64, or uint8. it wont be a list.